### PR TITLE
[T-6761][ADD] partner_children_address_autocreate

### PR DESCRIPTION
**Pull request by Tisho99**

### CODE REVIEW

**Review by sara-castello — APPROVED** [review]
functional review LGTM

File: partner_children_address_autocreate/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+from odoo import _, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        if not self.env.context.get("skip_partner_children_address_autocreate"):
```
Comment (manuelregidor) [inline partner_children_address_autocreate/models/res_partner.py:12]: I think you should mention this in the README file.

**Comment by Tisho99** [comment]
@manuelregidor 

Can you re review?

I have changed the readme

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Comment by Tisho99** [comment]
@manuel-florido 

Can you check the unreleased dependencies error?

**Comment by manuel-florido** [comment]
@Tisho99 

You need to remove the content of test-requirements.txt, it needs to be empty

**Comment by Tisho99** [comment]
@ValentinVinagre Can you merge?

**Review by ValentinVinagre — APPROVED** [review]

**Pull request merged**